Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM python:3.7
 
-ADD ci ci
+ADD minimum_dependency_generator minimum_dependency_generator
 ADD requirements.txt requirements.txt
+ADD test-requirements.txt test-requirements.txt
```

Comment:
I don't think we need the test requirements in the container

Reply:
Fixed